travis-ci/install.sh

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
-# setup for node 6
+# setup for node 7
 sudo rm -rf ~/.nvm
-curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
```

Comment:
yep AFAIK the `setup_6.x` script configures the PPA to stick on the latest LTS.
I'd propose to create an env variable (`NODE_VERSION` for instance) in the `.travis.yml`, so we can check against Node.js 6.x and Node.js 7.x?